src/serlo-editor/plugins/text/components/text-editor.tsx

```diff
@@ -54,6 +54,11 @@ export function TextEditor(props: TextEditorProps) {
     [createTextEditor]
   )
 
+  const mayManipulateSiblings = useMemo(
+    () => selectMayManipulateSiblings(store.getState(), id),
+    [id]
+  )
```

Comment:
Thanks for the question, was not 100% sure here either.
As I understand it, drag&dropping plugins is a feature that only works for children of rows plugins so I'm pretty sure this can't change right now.